### DISCUSSION TITLE
Lasted updates

### DISCUSSION THREAD

**Discussion by mart-mac** [wills106/homeassistant-solax-modbus#361] [General]
Probably a really simple question,  but what is the correct way to add the latest updates to the integration.  I've tried just copying over the plugin_solis.py file, but this comes up with error.  Do I need to remove it all and add it again?

**Comment by alienatedsec** [comment]
The latest changes `backup` related are merged but not yet released. However, you need to load beta versions in HACS or copy the entire folder to the custom components.

**Reply by alienatedsec** [reply]
@mart-mac 
![image](https://user-images.githubusercontent.com/73167064/226413032-27584907-dc2d-486b-ae0f-1c4ba08fa169.png)

![image](https://user-images.githubusercontent.com/73167064/226413230-534c6d74-648a-48c1-af0c-1f379ef976d6.png)
You can also load the main version if you wish - that should have the above backup/reserve mode integrated
